scripts/schema_to_dataclass.py

```diff
@@ -0,0 +1,166 @@
+#!/bin/env/python
+import sys
+import re
+import json
+from pathlib import Path
+
+map_schema_type_to_python = {
+    "object": "Dict",
+    "array": "List",
+    "integer": "int",
+    "string": "str",
+    "number": "int",
+    "boolean": "bool",
+    "any": "Any",
+}
+
+
+def create_dataclass(name):
```

Comment:
it is worth it to wrap the dataclass into the create_dataclass when we could just use directly dataclass()?

Reply:
I think the script can be improved in several ways. This PR isn't about the script, more about the classes it generates. 

I added the script in order not to lose it. For this PR I ignore comments on the script if you don't mind.

Reply:
okidoki ;)